cli/dcoscli/data/config-schema/package.json

```diff
@@ -5,7 +5,8 @@
     "sources": {
       "type": "array",
       "items": {
-        "type": "string"
+        "type": "string",
+        "pattern": "^(?:(?:https?)://)(?:(?:[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?\\.)+(?:[a-zA-Z]{2,6}\\.?|[a-zA-Z0-9-]{2,}\\.?)|localhost|\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\\.\\d{1,3})(?::\\d+)?(?:/?|[/?]\\S+)$"
```

Comment:
We need more varied validation here - in particular, package sources can be either a Git URI, HTTP/HTTPS URI or a file URI (see the difference sources here: https://github.com/mesosphere/dcos-cli/blob/master/dcos/package.py#L727).